src/core/caniuse.js

```diff
@@ -43,6 +43,12 @@ const supportTitles = new Map([
   ["d", "Disabled by default (needs to enabled)."],
 ]);
 
+const link = createResourceHint({
```

Comment:
Might be good to only add this if there isn't one already. So, maybe query the head for an existing one?